hirola/front/templates/front/dashboard.html

```diff
@@ -311,6 +324,14 @@ <h5>My Reviews</h5>
     </div>
 </div>
 </div>
+<script type="text/javascript">
+        var success_messages = []
```

Comment:
Put all javascript in a javascript file and not on the HTML files. It is not best practice.

Reply:
I'm using this script to pass the django arguments to js, I'm still looking for a more convinient way

Reply:
Okey-dokey